server/services/store/sqlstore/migrate.go

```diff
@@ -59,14 +60,14 @@ func (s *SQLStore) getMigrationConnection() (*sql.DB, error) {
 		}
 	}
 
-	db, err := sql.Open(s.dbType, connectionString)
-	if err != nil {
-		return nil, err
+	var settings mmModel.SqlSettings
```

Comment:
Nit: we're not finished yet but but trying to standardize on `mm_model` since package names are lowercase only by convention.  We can change this when we do a final pass across the app.